src/mscorlib/corefx/System/Globalization/CultureInfo.Unix.cs

```diff
@@ -13,14 +13,20 @@ public partial class CultureInfo : IFormatProvider
         /// </remarks>
         private static CultureInfo GetUserDefaultCultureCacheOverride()
         {
-            // TODO: Implement this fully.
             return null;
         }
 
-        private static CultureInfo GetUserDefaultCulture()
+        internal static CultureInfo GetUserDefaultCulture()
         {
-            // TODO: Implement this fully.
-            return CultureInfo.InvariantCulture;
+            CultureInfo temp = null;
```

Comment:
Could we default this to Invariant?  It worries me that we return null here (I understand that CultureData.GetDefaultLocaleName should never not return a localeName, (unless there is an ICU problem) but we might as well be defensive.